src/assets/logo.svg

```diff
@@ -0,0 +1 @@
563-1.776-1.774zm-10.166-8.39c0 5.164-3.952 11.133-9.52 11.133-3.79 0-5.97-2.502-5.97-5.648 0-6.21 9.683-6.856 13.796-6.856h1.693v1.37zM74.54 11.805c-4.844 0-8.07 1.614-10.25 3.47-1.21 1.047-2.257 2.177-2.98 3.468h-.164c.08-.162.08-.323.08-.566v-1.533c0-2.178-1.29-3.872-5.08-3.872h-10.57v7.664h4.6c1.21 0 1.774.565 1.774 1.774v40.174h-5.97v7.664h22.185v-7.664h-5.97v-9.762c0-1.048 0-1.936-.08-2.58l-.164-.97h.164c.807 1.05 1.775 2.018 2.905 2.905 1.934 1.532 4.84 2.904 8.95 2.904 10.812 0 18.798-8.47 18.798-21.538 0-12.744-7.098-21.537-18.23-21.537zM72.116 46.25c-6.775 0-10.244-6.29-10.244-12.665 0-9.035 4.92-12.987 10.326-12.987 6.128 0 10.243 5.082 10.243 12.907 0 8.148-4.68 12.746-10.326 12.746zM116.973 11.805c-12.342 0-20.895 8.874-20.895 21.54 0 11.616 8.47 21.538 22.104 21.538 5.162 0 9.277-1.372 12.182-2.904 1.694-.89 3.228-1.776 4.436-2.825l-3.952-7.26c-1.048.806-2.177 1.532-3.47 2.178-2.176 1.21-5 2.178-8.47 2.178-6.212 0-11.695-3.953-12.34-11.454h28.476l.16-1.29c.082-.727.082-1.615.082-2.18 0-10.89-6.456-19.52-18.313-19.52zm-10.165 16.94c.97-5.485 4.68-9.034 9.92-9.034 4.44 0 7.745 3.31 7.99 9.036h-17.91z"/><path fill="#00CFB5" d="M201.272 51.98c1.694-.89 3.228-1.776 4.438-2.825l-3.952-7.26c-1.05.806-2.18 1.532-3.47 2.178-2.18 1.21-5.002 2.178-8.47 2.178-6.212 0-11.698-3.953-12.344-11.454h28.477l.16-1.29c.083-.727.083-1.615.083-2.18 0-10.89-6.452-19.52-18.312-19.52-12.342 0-20.893 8.873-20.893 21.538 0 11.617 8.47 21.54 22.104 21.54 5.166 0 9.278-1.373 12.182-2.905zm-13.63-32.27c4.435 0 7.743 3.308 7.985 9.035h-17.91c.97-5.485 4.68-9.035 9.924-9.035z"/><path fill="#838386" d="M162.23 45.767c-2.905 0-8.23-.888-8.23-7.986V20.438h9.357v-7.664H154V1.56h-6.018c-.634 6-4.877 10.912-10.518 12.544v6.333h6.292V38.83c0 13.793 11.373 15.488 17.183 15.488.965 0 1.692 0 2.257-.08l.888-.162v-8.39h-.485c-.32 0-.807.08-1.37.08z"/><path fill="#00CFB5" d="M255.656 28.486H265.9v-8.05h9.356v-7.663H265.9V1.56h-6.02c-.633 6-4.876 10.912-10.517 12.544v6.333h6.292v8.05zM275.5 45.687c-.322 0-.808.08-1.372.08-2.904 0-8.228-.888-8.228-7.986v-3.076h-10.244v4.125c0 4.365 1.144 7.513 2.886 9.785-4.247 19.656-21.93 33.915-42.083 33.915-12.503 0-24.372-5.427-32.562-14.89l-7.116 6.16c9.98 11.528 24.44 18.14 39.677 18.14 23.495 0 44.228-15.914 50.51-38.273 2.146.503 4.23.65 5.868.65.967 0 1.694 0 2.26-.08l.887-.16v-8.39h-.486z"/><path fill="#00CFB5" d="M232.174 28.503c-6.615 0-23.154 1.29-23.154 13.876 0 8.388 6.533 12.503 13.635 12.503 4.84 0 7.904-1.937 10.084-3.953 1.207-1.21 2.176-2.5 2.9-3.952h.163c0 .16 0 .32-.08.564-.082.403-.16.888-.16 1.532v.484c0 2.34 1.45 4.355 5.24 4.355h9.683V46.25h-3.71c-1.21 0-1.775-.564-1.775-1.775V28.02c0-10.567-5.89-16.215-16.94-16.215-4.92 0-9.036 1.21-11.86 2.5-1.692.726-3.145 1.614-4.435 2.582l3.79 7.1c1.05-.726 2.26-1.452 3.55-2.017 2.18-.968 4.922-1.936 8.067-1.936 4.275 0 7.584 1.855 7.584 7.098v1.37h-2.582zm2.66 6.212v1.372c0 5.163-3.952 11.132-9.52 11.132-3.79 0-5.97-2.502-5.97-5.648 0-6.21 9.683-6.856 13.796-6.856h1.694z"/></svg>
```

Comment:
El logo en svg creo que esta mejor en el static. En assets lo compila webpack, mientras que en static lo sirve nuxt directamente

Reply:
I'm not working with nuxt. 
Is it the same for Vue?